src/groups/mwc/mwcu/mwcu_blobiterator.t.cpp

```diff
@@ -274,10 +273,8 @@ static void test3_forwardIterator()
             ASSERT_NE_D("line " << data.d_line, iter2, end);
             const mwcu::BlobPosition p(iter2.position());
             const mwcu::BlobIterator it(iter2++);
-            bsl::iterator_traits<mwcu::BlobIterator>::reference r(*it);
-            char                                                c;
+            char                     c;
             mwcu::BlobUtil::readNBytes(&c, blob, p, 1);
-            ASSERT_EQ_D("line " << data.d_line, c, r);
```

Comment:
You should keep this assertion. Since `r` was initialized as `::reference r(*it);`, please check whether 

```
ASSERT_EQ_D("line " << data.d_line, c, *it);
```

makes sense.

Reply:
To make this change, I had to return iterator typedefs